haystack/preview/pipeline.py

```diff
@@ -23,9 +23,9 @@ class Pipeline(CanalsPipeline):
 
     def __init__(self):
         super().__init__()
-        self.stores = {}
+        self.stores: Dict[str, Store] = {}
```

Comment:
I think the stores should be part of the private interface of `Pipeline`, I see no reason to have accesor methods otherwise.

Reply:
I don't see much harm at leaving them accessible either, but no strong opinions. I'll turn it private.